pkg/kubelet/config/file.go

```diff
@@ -200,3 +201,21 @@ func (s *sourceFile) replaceStore(pods ...*v1.Pod) (err error) {
 	}
 	return s.store.Replace(objs, "")
 }
+
+func checkPlainTextFile(file string) (bool, error) {
+	out, err := exec.Command("file", "--mime-type", file).CombinedOutput()
```

Comment:
Thanks @dixudx for your example and suggestion! That's Great! 

However, as per @mtaufen 's comment, restricting users to provide `text/plain` manifest might not be okay.